modules/mobile/docs/openapi.json

```diff
@@ -13005,6 +13005,16 @@
                   "claimTypeCode": {
                     "type": "string",
                     "example": "020NEW"
+                  },
+                  "displayTitle": {
+                    "type": "string",
+                    "description": "Formatted title for display",
+                    "example": "Claim for compensation"
+                  },
+                  "claimTypeBase": {
+                    "type": "string",
+                    "description": "Base type of claim",
+                    "nullable": true
```

Comment:
Unless I'm misreading this, I believe `displayTitle` is nullable, not `claimTypeBase`. Same goes for the instances below.